package.json

```diff
@@ -11,7 +11,8 @@
     "lab": "5.x.x"
   },
   "scripts": {
-    "test": "make test-cov",
+    "test": "node node_modules/lab/bin/lab -a code -t 100 -L",
```

Comment:
I know it, I follow [hapi package.json](https://github.com/hapijs/hapi/blob/master/package.json#L53-L55)
Do you think that there is reason for the `node_modules` path there?